README.smbconf.md

```diff
@@ -53,11 +53,15 @@ mapistore named properties backend
 mapistore indexing backend
 --------------------------
 
-- __mapistore:indexing_backend = STRING__ This option specified the
+- __mapistore:indexing_backend = STRING__ This option specifies the
   database URL to be used for indexing backend. If not present LDB
   backend will be use. The URL must have a format like:
   mysql://user:pass@hostname/database
 
+- __mapistore:indexing_cache = STRING__ This option specifies the
+  connection string to use to connect to the memcached server used
+  for fmid caching.
```

Comment:
We should add both an example and a link to the doc about how this connection string should be (http://docs.libmemcached.org/libmemcached_configuration.html).